spec/features/edit_a_pom_spec.rb

```diff
@@ -44,22 +45,36 @@
     expect(page).to have_content('Select number of days worked')
   end
 
-  it "makes an inactive POM active", vcr: { cassette_name: 'prison_api/edit_poms_activate_pom_feature' } do
-    # This doesn't do what it appears to - the URL is wrong so we're not editing an inactive POM... :-(
-    visit "/prisons/LEI/poms#inactive"
-    click_link 'Moic Integration-Tests'
+  describe "making an inactive POM active", vcr: { cassette_name: 'prison_api/edit_poms_activate_pom_feature' } do
+    let(:prison) { Prison.find('LEI') }
+    let(:other_prison) { create(:prison) }
 
-    click_link "Edit profile"
+    before do
+      # This is the ID for MOIC Integration-tests
+      create(:pom_detail, :inactive, prison: other_prison, nomis_staff_id: 485_758)
+      create(:pom_detail, :inactive, prison: prison, nomis_staff_id: 485_758)
+    end
```

Comment:
I can't see where other_prison is being used @starswan. What is its purpose? I think that might be just a question for me. :)